great-idea/css/index.css

```diff
@@ -63,4 +63,263 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+ol, ul {
+	list-style: circle;
+}
+
+.main-content {
+	margin: 0 25%;
+}
+
+header {
+	display: flex;
+	margin: 5% 5% 5% 0;
+}
+
+nav {
+	display: flex;
+	justify-content: space-evenly;
+	width: 80%;
+}
+
+nav a {
+	margin-top: 1%;
+	text-decoration: none;
+	color: grey;
+}
+
+/* .logo {
+	margin-right: 3%;
+} */
+
+.top-feature {
+	display: flex;
+	justify-content: space-evenly;
+	margin: 5% 10%;
+}
+
+.feature-text {
+	width: 30%;
+	height: 220px;
+	margin: 5% 10%;
+}
+
+.feature-text h2 {
+	font-size: 70px;
+	display: flex;
+	justify-content: center;
+}
+
+.get-started-btn {
+	margin-left: 25%;
+	width: 50%;
+	height: 15%;
+	background-color: #fff;
+}
+
+.top-content {
+	display: flex;
+	margin: 4% 0;
+	justify-content: space-around;
+}
+
+.features {
+	width: 45%;
+}
+
+.about {
+	width: 45%;
+}
+
+.middle-image {
+	display: flex;
+	justify-content: center;
+}
+
+.bottom-content {
+	display: flex;
+	justify-content: space-around;
+	margin: 4% 0;
+}
+
+.services {
+	width: 30%;
+}
+
+.product {
+	width: 30%;
+}
+
+.vision {
+	width: 30%;
+}
+
+.main-address {
+	display: flex;
+	flex-direction: column;
+
+}
+
+.main-phone {
+	margin: 2% 0;
+}
+
+.copyright {
+	display: flex;
+	justify-content: center;
```

Comment:
Remember, flex manipulates its _children elements_, not itself. To center the text here, you should instead use `text-align: center;`. You may also need to make this element `display: block;` so that it's on its own line. (_They are rendered side-by-side on my end_)

Reply:
Thats weird. This centered it for me in chrome. Normally I would have used text-align, but I was trying to implement flexbox as much as possible.